ztest/ztest.go

```diff
@@ -504,7 +505,7 @@ func runzq(path, zedProgram, input string, outputFlags []string, inputFlags []st
 		// tests.
 		return outbuf.String(), errbuf.String(), err
 	}
-	proc, err := compiler.ParseOp(zedProgram)
+	proc, err := compiler.Parse(zedProgram)
```

Comment:
Just do this so we don't need to thread the compiler down through all these calls (or even make it part of the ztest interface)?
```suggestion
	proc, err := zruntime.NewCompiler().Parse(zedProgram)
```